<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelEdit</class>
 <widget class="QDialog" name="ModelEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="tabModelSetup">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="modelNameLE">
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Timer Mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="timerModeCB"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Timer Direction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="timerDirCB">
         <item>
          <property name="text">
           <string>Count Down</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Count Up</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Timer Value</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTimeEdit" name="timerValTE">
         <property name="currentSection">
          <enum>QDateTimeEdit::MinuteSection</enum>
         </property>
         <property name="displayFormat">
          <string>mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Trim Increment</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="trimIncCB">
         <item>
          <property name="text">
           <string>Exponential</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Extra Fine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Coarse</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Throttle Trim</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="thrTrimChkB">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Throttle Expo</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="thrExpoChkB">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>instaTrim Switch</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="trimSWCB"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Center beep</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="bcRUDChkB">
           <property name="text">
            <string>RUD</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="bcELEChkB">
           <property name="text">
            <string>ELE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="bcTHRChkB">
           <property name="text">
            <string>THR</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="bcAILChkB">
           <property name="text">
            <string>AIL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="bcP1ChkB">
           <property name="text">
            <string>P1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QCheckBox" name="bcP2ChkB">
           <property name="text">
            <string>P2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QCheckBox" name="bcP3ChkB">
           <property name="text">
            <string>P3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Pulse Shift (polarity)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="pulsePolCB">
         <item>
          <property name="text">
           <string>Positive</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Negative</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Protocol</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="protocolCB">
         <item>
          <property name="text">
           <string>PPM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Silver A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Silver B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Silver C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Trac 09</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="numChannelsSB">
         <property name="suffix">
          <string>ch</string>
         </property>
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ppm Delay</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="ppmDelaySB">
         <property name="suffix">
          <string> usec</string>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>800</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExpo">
      <attribute name="title">
       <string>Expo/Dr</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabMix">
      <attribute name="title">
       <string>Mixes</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabLimit">
      <attribute name="title">
       <string>Limits</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabCurve">
      <attribute name="title">
       <string>Curves</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabSwitch">
      <attribute name="title">
       <string>Switches</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabTrims">
      <attribute name="title">
       <string>Trims</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QSlider" name="slider_S2">
         <property name="minimum">
          <number>-127</number>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,1,0,0,1">
         <item row="2" column="0">
          <widget class="QSpinBox" name="spinBox_S2">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-127</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QSpinBox" name="spinBox_S1">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-127</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Label_S1">
           <property name="text">
            <string>STICK 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Label_S2">
           <property name="text">
            <string>STICK 2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,0,0,1,0,0,1">
         <item row="2" column="0">
          <widget class="QSpinBox" name="spinBox_S3">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-127</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QSpinBox" name="spinBox_S4">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-127</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Label_S4">
           <property name="text">
            <string>STICK 4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Label_S3">
           <property name="text">
            <string>STICK 3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="slider_S3">
         <property name="minimum">
          <number>-127</number>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="slider_S1">
         <property name="minimum">
          <number>-127</number>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="slider_S4">
         <property name="minimum">
          <number>-127</number>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_S3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_S3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_S4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_S4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_S1</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_S1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_S2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_S2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_S2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_S2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_S3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_S3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_S4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_S4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_S1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_S1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
